prometheus/CMakeLists.txt

```diff
@@ -3,13 +3,19 @@ cmake_minimum_required(VERSION 2.8)
 project("prometheus-client")
 set(PROJECT_VERSION "0.99")
 set(PROJECT_DESCRIPTION "Prometheus C++ client library")
+set(CMAKE_MACOSX_RPATH true)
 
 include(GNUInstallDirs)
 include(FindPkgConfig)
+if (APPLE)
+  # Find homebrew's libicu on Mac OS although it is hidden pretty well
+  set(ENV{PKG_CONFIG_PATH} "$ENV{PKG_CONFIG_PATH}:/usr/local/opt/icu4c/lib/pkgconfig")
```

Comment:
> Is this really how the best-practice is for homebrew?

Oh, and no. Homebrew usually works like a charm. This is a rare exception. Here is what is says when installing icu4c:

````
==> Pouring icu4c-57.1.el_capitan.bottle.tar.gz
==> Caveats
This formula is keg-only, which means it was not symlinked into /usr/local.

OS X provides libicucore.dylib (but nothing else).

Generally there are no consequences of this for you. If you build your
own software and it requires this formula, you'll need to add to your
build variables:

    LDFLAGS:  -L/usr/local/opt/icu4c/lib
    CPPFLAGS: -I/usr/local/opt/icu4c/include
    PKG_CONFIG_PATH: /usr/local/opt/icu4c/lib/pkgconfig

==> Summary
🍺  /usr/local/Cellar/icu4c/57.1: 265 files, 65.0M
````